src/hotspot/share/utilities/copy.hpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2003, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2003, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I forgot to undo year change in this file.

Reply:
Yup. Done.